### PR TITLE
1 create a hello world program

### LINKED ISSUE #2: Develop a hello world program

### PR DESCRIPTION
**Pull request by ColinBindi**

### CODE REVIEW

**Comment by ColinBindi** [comment]
I want to add hello world

File: hello-world.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class HelloWorld {
```
Comment (mopeters8) [inline hello-world.java:1]: this is beautiful!!!

File: hello-world.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    public static void main(String[] args) {
+        System.out.println("Hello, World!");
```
Comment (mopeters8) [inline hello-world.java:4]: add a comment!

**Review by mopeters8 — CHANGES_REQUESTED** [review]
plz add a comment

**Review by mopeters8 — APPROVED** [review]

**Pull request merged**